package.json

```diff
@@ -5,13 +5,20 @@
     "type": "git",
     "url": "https://github.com/axelarnetwork/axelar-cgp-sui"
   },
+  "files": [
+    "scripts",
+    "move",
+    "!move/**/dependencies/**/*.{mv,move,mvsm}",
+    "!move/**/source_maps/**/*.{mv,move,mvsm}"
```

Comment:
We don't need to check in a build. The deployment script will actually build it, since Move hardcodes the dependency addresses on compilation. So pre-build has the wrong addresses.